pkg/driver/async_processing.go

```diff
@@ -0,0 +1,119 @@
+package driver
+
+import (
+	"bytes"
+	log "github.com/sirupsen/logrus"
+	"github.com/vhive-serverless/loader/pkg/driver/clients"
+	"io"
+	"math"
+	"net"
+	"net/http"
+	"strconv"
+	"sync"
+	"time"
+)
+
+func (d *Driver) writeAsyncRecordsToLog(logCh chan interface{}) {
```

Comment:
I am not a huge fan of having a chan interface. Can you specify a type here? I assume the log as a homogeneous type.